examples/simplesample_http/simplesample_http.ino

```diff
@@ -130,8 +117,6 @@ void initTime() {
     Serial.print("Fetched NTP epoch time is: ");
     Serial.println(epochTime);
 
-    iotHubClient.setEpochTime(epochTime);
```

Comment:
epochTime is now an unused variable.  Is there any reason to retrieve this from the NTP server if we're not using this?